mqtt/mqtt-bridge/src/client.rs

```diff
@@ -431,9 +522,12 @@ pub enum ClientError {
     #[error("failed to shutdown custom mqtt client: {0}")]
```

Comment:
nit: update all error messages to include "{0}" where there is a #[from]

Reply:
Actually I was thinking about the opposite: remote "{0}" from each error. Wherever we need a full chain of error we should rely on the framework (tracing) to log with a full stack. What do you think?

Reply:
I don't think it works that way right now, or maybe we have wrong setup somewhere. In my manual tests I couldn't get the full error logs that's why I suggested to add {0} everywhere.